tests/Avalonia.Controls.UnitTests/Presenters/TextPresenter_Tests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Avalonia.Controls.Presenters;
+using Xunit;
+
+namespace Avalonia.Controls.UnitTests.Presenters
+{
+    public class TextPresenter_Tests
+    {
+        [Fact]
+        public void TextPresenter_can_contain_null_with_password_char_set()
```

Comment:
Nit: to be consistent with the rest of the tests, this should be capitalized as `TextPresenter_Can_Contain_Null_With_Password_Char_Set` (as with the rest of the tests here).